tests/end-to-end/tools/example-index-generator/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"text/template"
+)
+
+func main() {
+	dir := os.Getenv("EXAMPLES_DIRECTORY")
+	if dir == "" {
+		panic("Missing `EXAMPLES_DIRECTORY` environment variable")
+	}
+
+	files, err := ioutil.ReadDir(dir)
+	if err != nil {
+		panic(err)
+	}
+
+	data := make([]Data, 0)
+	for _, f := range files {
+		if !strings.HasSuffix(f.Name(), ".graphql") {
+			continue
+		}
+		withoutExt := strings.Replace(f.Name(), ".graphql", "", -1)
+		withoutDash := strings.Replace(withoutExt, "-", " ", -1)
```

Comment:
I believe that previously the names was capitalized. I think it is a nice idea, but it's your decision.